app/presenters/finder_content_item_presenter.rb

```diff
@@ -45,10 +45,11 @@ def details
       document_noun: schema.fetch("document_noun"),
       document_type: metadata.fetch("format"),
       email_signup_enabled: metadata.fetch("signup_enabled", false),
-      format_name: metadata.fetch("format_name"),
+      filter: metadata.fetch("filter", {}),
+      format_name: metadata.fetch("format_name", nil),
```

Comment:
This change (adding the `nil` fallback) doesn't seem to relate to the commit message.